content/events/2021-online-unconference/_index.md

```diff
@@ -26,26 +26,30 @@ who are or identify as RSEs, but anybody interested in the interface of research
 
 ## What contributions we welcome
 
-In principle we welcome any contributions which respect the [Code of Conduct](./code-of-conduct)
-and in principle of any length. We will try to arrange a program based on your input.
+We welcome *any kind of contribution* which respects the [Code of Conduct](./code-of-conduct). 
+We will try to arrange a program based on your input.
 
 Here are some examples of possible contributions:
-- Discussion topics (about software, the research software community, or anything related to RSEs)
-- Demonstrations (tools you like, software you wrote)
-- Quick workshops or [ReproHacks](https://reprohack.github.io/reprohack-hq/) (only 3 hours in a day)
-- Talks 
-- Workshops and lessons: teach us something new
-- Can also be something unfinished and almost working where you would like to get feedback on
+- *Discussion topics*: is software, research software community or research software engineer something close to your heart? Come chatting with other enthusiasts!
+- *Demonstrations:* show us some tools or softwares that you like
+- *Workshop or [ReproHacks](https://reprohack.github.io/reprohack-hq/)* (only 3 hours in a day)
+- *Talks*: teach us something new 
+- *On going projects*: share with others something unfinished and almost working where you would like to get feedback on
+- *Something else*: something you would like to share with others that does not fit any of the previuous points? We are looking forward to hearing about it!
 
 
 ## How to contribute ideas
 
-- Come to the meeting and pitch your discussion idea, talk or workshop.
-- Get support and feedback on your idea before the event by submitting
-  it to as a [GitHub issue](https://github.com/nordic-rse/nordic-rse.github.io/issues/new).
-- You can also email us at <nordic-rse-organizers@neic.no>.
-- Note that we will publish name and abstracts of contributions.
-- Contributions should not violate [Code of Conduct](./code-of-conduct).
+
+- Would like to participate but don't really know how? Chat with us in the [Zulip chat](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse), we will figure out something together.
+- Once you have an idea for your contribution, open a github issue [here](https://github.com/nordic-rse/nordic-rse.github.io/issues/new/choose), use the *Unconference contribution* template. 
+- You can also email us at <nordic-rse-organizers@neic.no>. In this case, please try to follow [this template](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md)
+- Note that you don't need to submit a fully polished proposal, you can update your idea until the unconference and you will receive all the feedback and support that you need.
+- For organisation purposes, we kindly ask you to submit your idea by June 28th 2021. 
+- If you miss the deadline, you are still welcome to join the unconference and add you idea on hackmd. **AND THEN WHAT?**
```

Comment:
Maybe something like:
If you miss the deadline, you still have the chance to add a contribution live during the event ("Birds of a Feather").

Reply:
My fear is that that way it's not very clear "how" they will be able to contribute on the fly. My understanding is that knowing participants in advance will help precomputing how many BO rooms are needed, how many parallel sessions etc. Will people registering on the unconference day also get a BO room on the fly? What happens if on the unconference day 20 people mark they want to do a 30 minutes demonstration on hackmd?

Reply:
true that if they haven't submitted any idea by the day before it's unlikely they'll have anything ready on the unconference day, maybe I'm just overthinking it

Reply:
Your fear is not ungrounded, I feel the same way as you do. But from what I understand from @rantahar and ohers, this is a feature not a bug. Participants contribute on the fly in the initial planning/discussion sessions, in the style of "A birds of the Feather", by discussing over Zoom and taking notes on HackMD. In other words, a lot of the event is done live by the participants, rather than us planning everything in advance. We chose an unconference specifically for this reason: there is not enough personpower to do all the planning and organization in advance. As you point out, issues can arise and it might(will) get chaotic. If we have 20 people who want to contribute on the fly, we are very happy! It will be first come first serve to fill out the initial agenda, and once that is full, we will start adding parallel sessions with BO rooms as needed.